lib/tree.rb

```diff
@@ -1,38 +1,57 @@
 class NoApplesError < StandardError; end
 
 class Tree
-  attr_#fill_in :height, :age, :apples, :alive
+  attr_accessor :height, :age, :apples, :alive
 
-  def initialize
+  def initialize(age, apples=[], alive, height)
+    @age = age
+    @apples = apples
+    @alive = alive
+    @height = height
   end
 
-  def age!
+  def age! # I'm not sure why this has a bang
+    if @age == 50
+      @alive = false
+    elsif
+      @age = @age + 1
+    end
   end
 
-  def add_apples
+  def add_apples(number_of_apples)
+    @apples.concat([Apple.new('red', 9)] * number_of_apples)
   end
 
   def any_apples?
+    @apples.length >= 1 ? true : false
```

Comment:
The ternary here is redundant. `@apples.length >= 1` already returns `true` or `false`.

`@apples.any?` should do the trick, too.